src/content/docs/about/getting-started.mdx

```diff
@@ -0,0 +1,273 @@
+
+## Setting a Target
+
+Once you have installed a Provider, you can set a [Target](/configuration/providers#add-a-target) to define how Daytona manages and deploys your development environment. Configuring a Target allows you to tailor the development environment for your projects.
+
+A Target is a configuration that specifies where and how your development environment will be deployed. It is associated with a Provider and includes all the details Daytona needs to create and manage your development environment in a specific location, such as a local machine, cloud server, or remote Docker instance.
+
+Once your Target is set, Daytona will use it whenever you create a new development environment. You can manage multiple Targets, switch between them, or delete those you no longer need.
+
+For more details on available Providers and related commands, use the `daytona target` command to explore options and manage your configurations.
+
+<DocumentList>
+<DocumentListItem
+    title="Set a Target"
+    subtitle="Learn how to set a Target to define how Daytona manages your development environment."
+    href="/configuration/providers#managing-targets"
+    expanded
+  />
+</DocumentList>
+
+## Choosing the Default IDE
+
+Daytona allows you to connect to your development environment using a variety of [IDEs](/usage/ide), including popular options like [Visual Studio Code (VSCode)](/usage/ide#vs-code), JetBrains IDEs, and [Terminal SSH](/usage/ide#terminal-ssh). By setting the default IDE, you can automatically open your development environment in your preferred environment, streamlining your workflow and reducing setup time.
+
+<br />
+
+<CardGrid gridNumber={9} mobileGridNumber={2} colsMinWidth='64px' colsMinWidthMobile='98px'>
+ <ImageCard icon={VSlogo} link="/usage/ide#vs-code" imageAlt="Visual Studio logo" />
+ <ImageCard  imageDark={PClogo} link="/usage/ide#jetbrains-pycharm-professional" imageAlt="PyCharm logo" />
+ <ImageCard  imageDark={IJlogo} link="/usage/ide#jetbrains-intellij-idea-ultimate" imageAlt="IntelliJ logo" />
+    <ImageCard  imageDark={WSlogo} link="/usage/ide#jetbrains-webstorm" imageAlt="WebStorm logo" />
+    <ImageCard  imageDark={GOlogo} link="/usage/ide#jetbrains-goland" imageAlt="GoLand logo" />
+    <ImageCard  imageDark={CLlogo} link="/usage/ide#jetbrains-clion" imageAlt="CLion logo" />
+    <ImageCard  imageDark={RDlogo} link="/usage/ide#jetbrains-rider" imageAlt="Rider logo" />
+    <ImageCard  imageDark={RMlogo} link="/usage/ide#jetbrains-rubymine" imageAlt="RubyMine logo"/>
+</CardGrid>
+
+For more details on setting the default IDE and related commands, use the `daytona ide` command to explore options and manage your configurations.
+
+<DocumentList>
+<DocumentListItem
+    title="Set Default IDE"
+    subtitle="Learn how to set your default IDE to automatically open your development environment."
+    href="/configuration/git-providers"
+    expanded
+  />
+</DocumentList>
+
+## Creating your Workspace
```

Comment:
I suggest we move this above the Providers section. Configuring Providers and Targets is not important for the initial onboarding because we provide a default Docker target.